data/tooltips.txt

```diff
@@ -466,3 +466,6 @@ tip "solar power?"
 
 tip "no hyperdrive?"
 	`This ship has no hyperdrive. If it took off, it would be unable to leave this star system.`
+
+tip "no fuel?"
+	`This ship has not enough fuel. If it took off, it would be unable to leave this star system.`
```

Comment:
"has not enough fuel" -> "does not have the fuel capacity to make a hyperspace jump"